pkg/validator/validator_test.go

```diff
@@ -25,6 +25,34 @@ var testData = []struct {
 	{"validHcl", []byte(`key = "value"`), true, HclValidator{}},
 	{"invalidHcl", []byte(`"key" = "value"`), false, HclValidator{}},
 	{"multipleInvalidHcl", []byte(`"key1" = "value1"\n"key2"="value2"`), false, HclValidator{}},
+	{"validPlist", []byte(`<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These strings are really long - recommend moving to a separate variable

Reply:
Moved them to an embedded file. Now it's more readable.